corehq/apps/hqwebapp/models.py

```diff
@@ -147,4 +149,22 @@ class ServerLocation:
         },
     }
 
-    SUBDOMAINS = {env: server['subdomain'] for env, server in ENVS.items()}
+    STAGING_CONFIG = {
+        STAGING: {
+            'country_code': 'un',
```

Comment:
Should that be `us`?

Reply:
No, I was copying Evan's example on his `serverlocation-staging` branch. It doesn't have to be specifically `un`, but I believe he was looking for a flag that stood out as essentially a placeholder.